db/migrate/20200414143331_add_float_to_event_schedules.rb

```diff
@@ -0,0 +1,9 @@
+class AddFloatToEventSchedules < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :event_schedules, :event_start_time, :timestamp
+    remove_column :event_schedules, :event_end_time, :timestamp
+    add_column :event_schedules, :event_start_time, :float
```

Comment:
Thanks for updating the EventSchedule table to include a float rather than a timestamp!